core/extension/index.d.ts

```diff
@@ -49,11 +49,12 @@ export {
 	MessageBoxType,
 	MessageBoxIconType,
 	MessageBoxValue,
+	SelectionSetEventType,
```

Comment:
Any reason these moved from where they are on `master`? Did you re-run the generation?

Reply:
It was ran a while back: [this commit](https://github.com/iTwin/itwinjs-core/pull/3347/commits/ea00efe1a9ebaa7233a18cc1d99e449ebab077d2)

Reply:
probably best to merge in master again and re-run the generation to be safe.